README.md

```diff
@@ -1,352 +1,6 @@
-Obsidian surrounded by 6 lava sources has a chance of converting to lava.  
-Credits: Skyrising  
-* Type: `boolean`  
-* Default value: `false`  
-* Required options: `true`, `false`  
-* Categories: `EXTRAS`, `EXPERIMENTAL`, `FEATURE`  
-  
-## renewablePackedIce
-Multiple ice crushed by falling anvils make packed ice.  
-* Type: `boolean`  
-* Default value: `false`  
-* Required options: `true`, `false`  
-* Categories: `EXTRAS`, `EXPERIMENTAL`, `FEATURE`  
-  
-## renewableSand
-Cobblestone crushed by falling anvils makes sand.  
-* Type: `boolean`  
-* Default value: `false`  
-* Required options: `true`, `false`  
-* Categories: `EXTRAS`, `EXPERIMENTAL`, `FEATURE`  
-  
-## renewableWitherSkeletons
-Skeletons turn into wither skeletons when struck by lightning.  
-* Type: `boolean`  
-* Default value: `false`  
-* Required options: `true`, `false`  
-* Categories: `FEATURE`, `EXTRAS`  
-  
-## repeaterPriorityFix
-Quick pulses won't get lost in repeater setups.  
-Probably brings back pre 1.8 behaviour.  
-Fixes [MC-54711](https://bugs.mojang.com/browse/MC-54711).  
-* Type: `boolean`  
-* Default value: `false`  
-* Required options: `true`, `false`  
-* Categories: `EXTRAS`, `BUGFIX`, `EXPERIMENTAL`  
-  
-## scaffoldingDistance
-Max distance for scaffolding.  
-* Type: `int`  
-* Default value: `7`  
-* Suggested options: `2`, `3`, `5`, `7`  
-* Categories: `FEATURE`, `EXTRAS`  
-* Additional notes:  
-  * You must choose a value from 0 to 7  
-  
-## spiderJockeysDropGapples
-Gives Spider jockeys a specified chance to drop enchanted golden apples.  
-0 is the default setting, no enchanted golden apples will be dropped  
-* Type: `int`  
-* Default value: `0`  
-* Required options: `0`, `50`, `100`  
-* Categories: `EXTRAS`, `FEATURE`  
-* Additional notes:  
-  * You must choose a value from 0 to 100  
-  
-## straySpawningInIgloos
-Only strays spawn in igloos.  
-* Type: `boolean`  
-* Default value: `false`  
-* Required options: `true`, `false`  
-* Categories: `FEATURE`, `EXTRAS`  
-  
-## updateSuppressionCrashFix
-Fixes updates suppression causing server crashes.  
-* Type: `boolean`  
-* Default value: `false`  
-* Required options: `true`, `false`  
-* Categories: `BUGFIX`, `EXTRAS`  
-  
-## variableWoodDelays
-Variable delays on wooden components (buttons, pressure plates).  
-* Type: `boolean`  
-* Default value: `false`  
-* Required options: `true`, `false`  
-* Categories: `EXTRAS`, `FEATURE`  
-  
-## y0DragonEggBedrockBreaking
-Let dragon eggs break Y0 bedrock.  
-Requires dragonEggBedrockBreaking to be set to true.  
-* Type: `boolean`  
-* Default value: `false`  
-* Required options: `true`, `false`  
-* Categories: `EXPERIMENTAL`, `EXTRAS`  
-  
+Carpet Extra - Unofficial Fork
+============
 
+### IF YOU WANT CARPET EXTRA, GO HERE: https://github.com/gnembon/carpet-extra
+THIS IS A FORK OF CARPET EXTRA FOR MY TESTING PURPOSES. DO NOT USE THIS FORK. I HAVE ABSOLUTELY NO IDEA WHAT I AM DOING SO DON'T LOOK TO ME.
```

Comment:
Hm... No?

Reply:
yeah, sorry, that's my fork
imma undo that and put it into a branch (of my fork) unrelated to this pr
i fucked up, sorry

Reply:
ok, should be fixed now
sorry abt the mistake

Reply:
As a recommendation for the future, it's always better to make PRs from new branches, so they don't mess up and are easier to update with upstream.